source/fundamentals/connection/mongoclientsettings.txt

```diff
@@ -291,60 +295,6 @@ regardless of the type of MongoDB cluster its' a part of:
 
 .. _mcs-socket-settings:
```

Comment:
Issue: I think some of the anchors are in the wrong locations after content was moved around

Reply:
see the connectionpool anchor also

Reply:
I *think* this is because you were looking at a version before I made updates when Chris pointed that out. I checked and they all seem correct. Please let me know if I missed one.